marlowe-dashboard-client/src/Types.purs

```diff
@@ -28,59 +22,22 @@ type WebData
 type DecodedWebData
   = RemoteData (Either AjaxError (NonEmptyList ForeignError))
 
+-- TODO: I want to put this in Contract.Types, but then there's a cyclic dependency
+-- between Contract.Types and WalletData.Types. There might be a better organisation
+-- of the types that avoids this problem.
```

Comment:
Maybe we need a PAB module, and that could include some of the types that ended up in Execution